packages/account/src/Sections/Security/ConnectedApps/connected-apps-info.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { InlineMessage, Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { observer } from '@deriv/stores';
+
+const ConnectedAppsInfo = observer(() => (
+    <InlineMessage
+        type='information'
+        size='md'
+        message={
+            <div>
+                <Text as='h4' weight='bold' size='xs'>
+                    <Localize i18n_default_text='What are connected apps?' />
+                </Text>
+                <ol className='connected-apps__list--ordered_list'>
```

Comment:
will this ordered list take care of spacing as shown in design file?
![Screenshot 2023-10-11 at 1 22 50 PM](https://github.com/shaheer-deriv/deriv-app/assets/122449658/08caa185-8f9d-4095-9b12-f5fed6f85eb7)
i think you will have to use flexbox. you may refer the file `packages/account/src/Sections/Security/ConnectedApps/connected-apps-empty.tsx`

Reply:
This  spacing automatically taken care of by the inline message component so the ordered list can be used here